open3d_slam_ros/launch/mapping.launch

```diff
@@ -17,7 +17,7 @@
 	<!-- END OF ARGS -->	
 
 	<param name="/use_sim_time" value="$(arg use_sim_time)" />
-	<arg name="rosbag_filepath" 
+	<arg name="parameter_file_path" 
 		default="$(arg parameter_folder_path)/$(arg parameter_filename)" /> 
```

Comment:
rosbag filepath has already been defined here, and it doesn't make sense to assign it like this, whereas parameter_file_path was missing

Reply:
Thanks for spotting it, that's indeed a big typo